### ISSUE TITLE
CIDER support

### ISSUE THREAD

**Issue by cjhowe7** [martinklepsch/tenzing#85]
I think all that needs to be done for this is to have a `.dir-locals.el` file with the following content:

```
((nil . ((cider-boot-parameters . "dev"))))
```

This will make it start up the `boot dev` task to create a development REPL with `cider-jack-in`.

**Comment by martinklepsch** [comment]
Hey @cjhowe7 that sounds like a good suggestion - do you know if the value of cider-boot-parameters has any effect on what is being ran when you execute cider-connect?

Cheers

**Comment by cjhowe7** [comment]
No, I don't think so. Here's the doc string for `cider-boot-parameters`: "Params passed to boot to start an nREPL server via `cider-jack-in'."

**Comment by abhi18av** [comment]
Hey, so is this moving into the `master` soon? 

This would really be like ideal dev for `tenzing` :)

**Comment by cjhowe7** [comment]
I think this makes it slightly harder to transition a Tenzing project to have a backend. I know the intent is to start with frontend-only first, but once a backend is desired, this config is better replaced with `cljs-repl-env` from `boot-cljs-repl`. Not sure how this impacts the decision here, though.
